fixtures/sitetree.json

```diff
@@ -1,274 +1,298 @@
+    },
+    "model": "sitetree.treeitem",
+    "pk": 1
 },
 {
-  "fields": {
-    "parent": null,
-    "access_restricted": false,
-    "access_permissions": [],
-    "description": "",
-    "insitetree": true,
-    "hint": "",
-    "url": "cms_page \"venue/\"",
-    "inbreadcrumbs": true,
-    "title": "Venue",
-    "tree": 1,
-    "access_perm_type": 1,
-    "alias": null,
-    "sort_order": 2,
-    "inmenu": true,
-    "access_loggedin": false,
-    "hidden": false,
-    "urlaspattern": true,
-    "access_guest": false
-  },
-  "model": "sitetree.treeitem",
-  "pk": 2
+    "fields": {
+        "parent": null,
+        "access_restricted": false,
+        "access_permissions": [],
+        "description": "",
+        "insitetree": true,
+        "hint": "",
+        "url": "/#/venue/",
+        "inbreadcrumbs": true,
+        "title": "Venue",
+        "tree": 1,
+        "access_perm_type": 1,
+        "alias": null,
+        "sort_order": 2,
+        "inmenu": true,
+        "access_loggedin": false,
+        "hidden": false,
+        "urlaspattern": false,
+        "access_guest": false
+    },
+    "model": "sitetree.treeitem",
+    "pk": 2
 },
 {
-  "fields": {
-    "parent": null,
-    "access_restricted": false,
-    "access_permissions": [],
-    "description": "",
-    "insitetree": true,
-    "hint": "",
-    "url": "#",
-    "inbreadcrumbs": true,
-    "title": "Sponsors",
-    "tree": 1,
-    "access_perm_type": 1,
-    "alias": null,
-    "sort_order": 4,
-    "inmenu": true,
-    "access_loggedin": false,
-    "hidden": false,
-    "urlaspattern": false,
-    "access_guest": false
-  },
-  "model": "sitetree.treeitem",
-  "pk": 3
+    "fields": {
+        "parent": null,
+        "access_restricted": false,
+        "access_permissions": [],
+        "description": "",
+        "insitetree": true,
+        "hint": "",
+        "url": "#",
+        "inbreadcrumbs": true,
+        "title": "About",
+        "tree": 1,
+        "access_perm_type": 1,
+        "alias": null,
+        "sort_order": 3,
+        "inmenu": true,
+        "access_loggedin": false,
+        "hidden": false,
+        "urlaspattern": false,
+        "access_guest": false
+    },
+    "model": "sitetree.treeitem",
+    "pk": 3
 },
 {
-  "fields": {
-    "parent": null,
-    "access_restricted": false,
-    "access_permissions": [],
-    "description": "",
-    "insitetree": true,
-    "hint": "",
-    "url": "home",
-    "inbreadcrumbs": true,
-    "title": "Home",
-    "tree": 1,
-    "access_perm_type": 1,
-    "alias": null,
-    "sort_order": 1,
-    "inmenu": true,
-    "access_loggedin": false,
-    "hidden": false,
-    "urlaspattern": true,
-    "access_guest": false
-  },
-  "model": "sitetree.treeitem",
-  "pk": 4
+    "fields": {
+        "parent": 3,
+        "access_restricted": false,
+        "access_permissions": [],
+        "description": "",
+        "insitetree": true,
+        "hint": "",
+        "url": "/#/about/mission/",
```

Comment:
There are hashes in the URL because it changes from conference site to conference site (i.e. PyData San Francisco 2016 is at http://pydata.org/sfo2016, PyData DC 2016 is at http://pydata.org/dc2016, etc.).

Since we store the conference identifier in Ansible, it should be possible to change these static fixtures into templates and use Ansible to generate ones with correct URLs, but I have not done this yet.